### ISSUE TITLE
Email address isn't cleaned per Gravatars requirements

### ISSUE THREAD

**Issue by alxwrd** [bsoyka/gravify#17]
**Describe the bug**
The Gravatar implementation docs (https://en.gravatar.com/site/implement/hash/) has the following advice for hashes to be consistent with their API:

>    1. Trim leading and trailing whitespace from an email address
>    2. Force all characters to lower-case
>    3. md5 hash the final string

**To Reproduce**

```python
from gravify import Gravatar

x = Gravatar("bensoyka@icloud.com")
y = Gravatar("  BeNsOyKa@iCLOUD.com  ")

x.hash == "7246821a7bf0b1b37794b39cb08ee052"
y.hash == "836dbf5fd50f27788559fdf4cceca903"
```

**Comment by bsoyka** [comment]
Ah, yes, this is definitely an issue. I am going to let someone else try to do this as it is Hacktoberfest time right now.

**Comment by fleetcaptain** [comment]
Hello, hope your day is going well. I took a stab at this, please let me know if there are any edits I need to make to the PR. #19

**Comment by bsoyka** [comment]
Closed via #19 

I will be releasing this to PyPI tomorrow morning when I’m back at the computer. (@alxwrd)
